rosidl_generator_cpp/cmake/rosidl_generator_cpp_generate_interfaces.cmake

```diff
@@ -104,7 +104,7 @@ configure_file(
   "${_visibility_control_file}"
   @ONLY
 )
-list(APPEND _generated_msg_headers "${_visibility_control_file}")
+list(APPEND _generated_headers "${_visibility_control_file}")
```

Comment:
Actually, looking at this I think this whole block is in the wrong location.  That is, the `add_custom_command` in the previous block is the one that defines the outputs from the Python command.  But since this block is added *after* that, this will never be part of `_generated_headers` and thus will not be tracked when something changes.

So I'd say that we need this fix, but we also need to move this entire block up above the `add_custom_commands`.

Reply:
You're right. I've also moved it to the appropriate place (right after the other headers were generated).